Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -474,8 +516,15 @@ static void activate(GApplication* application, WPEToolingBackends::ViewBackend*
 
 #if ENABLE_WPE_PLATFORM
     if (auto* wpeView = webkit_web_view_get_wpe_view(webView)) {
+        auto* wpeToplevel = wpe_view_get_toplevel(wpeView);
+        if (windowWidth > 0 && windowHeight > 0)
+            wpe_toplevel_resize(wpeToplevel, windowWidth, windowHeight);
+        if (windowMaximized)
+            wpe_toplevel_maximize(wpeToplevel);
+        if (windowFullscreen)
+            wpe_toplevel_fullscreen(wpeToplevel);
```

Comment:
fullscreen and maximizez should be mutually exclusive.

Reply:
Fixed.

```
+    if (windowMaximized && windowFullscreen) {
+        g_printerr("You cannot specify both --maximized and --fullscreen, these options are mutually exclusive.");
+        return 1;
+    }
```